io_ogre/ogre/scene.py

```diff
@@ -595,15 +609,15 @@ def dot_scene_node_export( ob, path, doc=None, rex=None,
 
         if ob.data.use_diffuse:
             a = doc.createElement('colourDiffuse'); l.appendChild( a )
-            a.setAttribute('r', '%s'%ob.data.color.r)
-            a.setAttribute('g', '%s'%ob.data.color.g)
-            a.setAttribute('b', '%s'%ob.data.color.b)
+            a.setAttribute('r', '%6f' % ob.data.color.r)
```

Comment:
```suggestion
            a.setAttribute('r', '%3f' % ob.data.color.r)
```
colors are probably converted to `uchar` somewhere along the way, so more than %3f seems like a waste. At least blender displays them as %3f